README.md

```diff
@@ -61,6 +72,21 @@ If you want to configure the version of depcheck to use explicitly provide a ref
 
 Additionally, you are allowed to set a default value for the directories to ignore. For that use the `ignoreDirsDefault` property and hand over the names of the directories to ignore as an array. The default value is `['node_modules', 'bower_components']`.
 
+Note that if you use a custom version of depcheck you must pass any special parsers explicitly.
+
+```javascript
+const depcheck = require('gulp-depcheck');
```

Comment:
Can you please remove the second const, indent the line with six spaces, and add a comma to the end of the first line?